etc/segments.csv

```diff
@@ -0,0 +1,64 @@
+SEGMENT,REPLACEMENT
```

Comment:
Note that the segments.tsv is not ordered! This means, if you have a sequence sugh as "(" first, and later `')`, the latter will never be active.

Reply:
Sorry, I am not following. The new code applies the replacement on the entirety of the segment, with segments split on basis of the white-space, doesn't it? I refer to this: https://github.com/lexibank/pylexibank/blob/f59716ae584ae6b5f5f93097d06d5afe404a982a/src/pylexibank/cldf.py#L128

The single case for this correction is working as intended (from the `cldf/forms.csv`):

```
Kabiyari-77_sit-1,,Kabiyari,77_sit,tu(')na,tu(')na,t u ʔ n a,,,,
```

Reply:
This is entry 748 in the raw wordlist, segmented in source as `t u ( ') n a`

Reply:
Ah, I see. Nice code, @xrotwang.

Reply:
With the pre-aspirations now out, I suppose I can merge?